### PR TITLE
added extra option to create buterknife onclick methods against views

### PR DESCRIPTION
**Pull request by andyb129**
added extra plugin option to generate onClick butterknife methods to plugin :-)

### CODE REVIEW

**Comment by destil** [comment]
Hi, thanks for your contribution!

Can you please post some screenshots how does your implementation look in the UI?

File: src/com/avast/android/butterknifezelezny/model/Element.java [hunk 1]
```diff
@@ -7,108 +8,94 @@
...
-     *
-     * @return
-     */
-    public boolean checkValidity() {
-        Matcher matcher = sValidityPattern.matcher(fieldName);
-        isValid = matcher.find();
-
-        return isValid;
-    }
+	public String id;
```
Comment (destil) [inline src/com/avast/android/butterknifezelezny/model/Element.java]: Wrong formatting - we use spaces, not tabs.

File: src/com/avast/android/butterknifezelezny/form/EntryList.java [hunk 2]
```diff
@@ -108,120 +108,116 @@ protected void addButtons() {
...
-
-    protected class CancelAction extends AbstractAction {
-
-        public void actionPerformed(ActionEvent event) {
-            if (mCancelListener != null) {
-                mCancelListener.onCancel();
-            }
-        }
-    }
+		mHolderCheck = new JCheckBox();
```
Comment (destil) [inline src/com/avast/android/butterknifezelezny/form/EntryList.java]: Wrong formatting, we use spaces, not tabs.

**Comment by andyb129** [comment]
I've added an animated gif of the new onClick tick box.  Hope this helps! :-)

**Comment by destil** [comment]
Great! It's indeed useful. Please rename "Click" in the column to "OnClick" - it will be more clear this way. And also please fix the formatting according to [our codestyle](https://github.com/avast/android-styled-dialogs/blob/master/code-formatting-config.xml).

**Pull request merged**